h/static/scripts/group-forms/utils/api.ts

```diff
@@ -87,6 +87,14 @@ export class APIError extends Error {
     this.response = response;
     this.json = json;
   }
+
+  /**
+   * Property that is true if the API call failed because the operation was
+   * aborted.
+   */
+  get aborted() {
+    return this.cause?.name === 'AbortError';
+  }
```

Comment:
I made this a property to make it slightly more convenient to test `error.aborted` on an error that may or may not be an instance of `APIError`.